Repository/AbstractAssetsRepository.php

```diff
@@ -121,7 +121,7 @@ public function search($query, $mode = 0)
     /**
      * {@inheritdoc}
      */
-    public function whatProvides(Pool $pool, $name)
+    public function whatProvides(Pool $pool, $name, $bypassFilters = false)
```

Comment:
@francoispluchino, this PR equivalent to #205, but also includes a fixed unit test.
Sorry if I do not understand something. This is my first pull request.